ceph-releases/jewel/ubuntu/14.04/demo/entrypoint.sh

```diff
@@ -155,12 +155,7 @@ function bootstrap_osd {
     chown -R ceph. /var/lib/ceph/osd/${CLUSTER}-0
     ceph-osd ${CEPH_OPTS} -i 0 --mkfs --setuser ceph --setgroup ceph
     ceph ${CEPH_OPTS} auth get-or-create osd.0 osd 'allow *' mon 'allow profile osd' -o /var/lib/ceph/osd/${CLUSTER}-0/keyring
-    if [ $(hostname -s) == 'default' ]; then
-      echo "A hostname of 'default' as used by docker-machine conflicts with Ceph."
-      echo "Rename your machine to fix this issue."
-      echo "https://github.com/ceph/ceph-docker/issues/168"
-    fi
-    ceph ${CEPH_OPTS} osd crush add 0 1 root=default host=$(hostname -s)
+    ceph ${CEPH_OPTS} osd crush add 0 1 root=default host=localhost
```

Comment:
Using localhost will bring a lot of confusion from a Ceph perspective.
If all of your OSDs are running on the host localhost this might raise some issues, I'd like to keep your previous patch.